website/docs/docs/ai/managing.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Managing PactFlow AI
+sidebar_label: Managing PactFlow AI
+---
+
+## Usage Limits
+
+Access to the feature is limited by a credit system, where each test generated uses 1 credit. Usage is metered based on the number of tests generated per calendar month, and reset on the 1st day of each month.
+
+Credits are alotted to your account based on your plan:
+
+1. PactFlow developer plans: 10 credits per month (total) 
+2. Enterprise Plus or plans with Contract Testing Pro Addon: 30 credits per purchased user per month
+3. All other plans (Individual, Team and Enterprise Plans): 10 credits per month
+
+Note that credits are allocated to the organisation, not individual users.
+
+### Examples
+
+1. **Starter (Developer) Plan**
+
+    If you have a free PactFlow plan, you are entitled to 10 tests per month no matter how many active users there are. 
+    *NOTE: this is a legacy plan and is no longer available for signups*
+
+2. **Team Plan**
+
+    If you are on a Team 10 Plan (5 active users), you receive for 10 credits x 10 users for a total of 100 credits.
+
+3. **Enterprise Plan**
+
+    If you are on an Enterprise Plan with 50 users (42 active users), you receive for 30 credits x 50 users for a total of 1500 credits.
+
+### Consumption
+
+Each successfully generated test consumes a credit. If your organisation reaches its limit, you will see the following error:
+
+```
+ERROR: Client error: 403 Forbidden AI credits are exhausted.
+```
+
+:::note
+The limits and your usage against these limits are not currently visible on the subscription page. This will be added in due course.
+:::
```

Comment:
Not sure if this the spot, but you could mention that the current usage can be checked through the `pactflow-ai status` subcommand.

Reply:
Thanks. I think it would be a nice touch.